rubygem-algebrick/rubygem-algebrick.spec

```diff
@@ -5,8 +5,8 @@
 
 Summary: Algebraic types and pattern matching
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 0.7.3
-Release: 4%{?dist}
+Version: 0.7.4
```

Comment:
No need to update this, and it now doesn't match the version required by `dynflow`.